js/main.js

```diff
@@ -1,2 +1,29 @@
 /* global data */
 /* exported data */
+var photo = document.getElementById('images');
+var photoUrl = document.getElementById('photo-url');
+function newPhoto(event) {
+  photo.src = photoUrl.value;
+}
+
+document.addEventListener('input', newPhoto);
+
+var form = document.querySelector('form');
+var title = document.getElementById('title-name');
+var notes = document.getElementById('Notes');
+localStorage.setItem(data, data);
```

Comment:
You want to make sure that all local storage functionality is in the `data.js` file